extensions/wellinks/wellinksClient.ts

```diff
@@ -0,0 +1,41 @@
+import fetch from 'node-fetch'
+
+export class WellinksClient {
+  private readonly _apiUrl: string
+  private readonly _apiKey: string
+  constructor(apiUrl: string, apiKey: string) {
+    this._apiKey = apiKey
+    this._apiUrl = apiUrl
+  }
+
+  readonly memberListEvent = {
+    insert: async (
```

Comment:
totally preference, but when dealing with more than two or three input parameters, i tend to like to use a de-structured object as input. e.g.
```ts
insert: async ({
  foo,
  bar,
  baz,
}: {
  foo: string
  bar: string
  baz: string
}): ...
```

That pattern will help with autocompletion and code maintenance down the road